popsycle/synthetic.py

```diff
@@ -180,8 +194,9 @@ def write_galaxia_params(output_root,
 
 
 def perform_pop_syn(ebf_file, output_root, iso_dir,
-                    bin_edges_number = None, BH_kick_speed=100, NS_kick_speed=350,
-                    set_random_seed = False):
+                    bin_edges_number=None, BH_kick_speed=100,
+                    NS_kick_speed=350,
+                    overwrite=False, seed=None):
```

Comment:
This might cause problems when merging w/ birth_kicks branch, but I think we already know/expect that.

Reply:
Yup, I'll take care of fixing that upon merging